src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -39,7 +40,7 @@
 
     /** A pattern to find the string: "// violation". */
     private static final Pattern VIOLATION_PATTERN = Pattern
-            .compile(".*//\\s*violation(?:$|\\W+.*$)");
+            .compile(".*//\\s*violation:\\s(.*)$");
```

Comment:
`violation:`?  did we agree to have colon?

also this regex does not match `// violation` and `// violation this is the violation message`

Reply:
So which separator do we use to separate ```// violation``` and ```violation message```? Just a space?

Reply:
yes, why not

Reply:
done